modules/artifact/src/test/java/org/mule/runtime/module/artifact/builder/AbstractArtifactFileBuilder.java

```diff
@@ -230,5 +220,45 @@ private void buildBrokenJarFile(File tempFile) throws UncheckedIOException {
     }
   }
 
+  /**
+   * @return a collection with custom {@link ZipResource}s to add to the artifact file.
+   */
   protected abstract List<ZipResource> getCustomResources();
+
+  /**
+   * @return the descriptor loader for the artifact. May be null.
```

Comment:
`May be null` is redundant and misleading since the return type is an `Optional`